hack/testing-olm-upgrade/upgrade-common

```diff
@@ -397,15 +397,8 @@ discover_versions(){
   major_version="$(echo $full_version | cut -d'.' -f1)"
   minor_version="$(echo $full_version | cut -d'.' -f2)"
 
-  # we shouldn't run into this yet... maybe for 6.0 but we should like
-  # add the 'Replaces' spec field and use that for the previous_version
-  if [[ $minor_version -eq 0 ]]; then
-    log::info "Will be unable to calculate the previous_version since our minor version is 0"
-    exit 0
-  fi
-
```

Comment:
can you explain why you removed this? it seems like the comment is discussing the exact issue here.. i would think we should improve the logic rather than just remove it.

Reply:
I dropped after cherry-picking #664 [specific fix](https://github.com/openshift/elasticsearch-operator/pull/664/files#diff-6b288e96dd32a3258c3c816bf02dbede9bff3bb69b9fb302be75122a91d56404R404) for 5.0. Thus can keep this lines again till we reach 6.0.